composer.json

```diff
@@ -2,10 +2,10 @@
     "name": "altapay/magento2-subscriptions",
     "license": "MIT",
     "description": "AltaPay: Payments less complicated",
-    "version": "1.0.6",
+    "version": "1.0.7",
      "require": {
-       "php": "^7.3.0",
-       "altapay/magento2-community":"^3.2.3"
+       "php": "^7.0 || ^8.1",
```

Comment:
You are downgrading the php requirement. Are you sure this is correct?

Reply:
@emicha I have updated the minimum PHP version to 7.3.